### PR TITLE
Fix link for the volume migration page

### PR DESCRIPTION
**Pull request by alicefr**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:

```release-note

```

### CODE REVIEW

File: docs/storage/disks_and_volumes.md [hunk 1]
```diff
@@ -2131,7 +2131,7 @@ The following strategies are supported:
   * `Replacement`: the update volumes will be replaced upon the VM restart.
   * `Migration`: the update of the volumes will trigger a storage migration of
     the old volumes to the new ones. More details about volume migration can be
-    found in the [volume migration documentation](docs/storage/volume_migration.md).
+    found in the [volume migration documentation](https://kubevirt.io/user-guide/storage/volume_migration/).
```
Comment (dhiller) [inline docs/storage/disks_and_volumes.md:2134]: I'd rather prefer a relative link here
```suggestion
    found in the [volume migration documentation](/storage/volume_migration/).
```

Note: the resulting link from the preview looks like this

https://deploy-preview-821--kubevirt-user-guide.netlify.app/storage/volume_migration/

**Review by dhiller — CHANGES_REQUESTED** [review]
Hey, thanks for removing all that trailing whitespace :+1: 

One remark inline.

**Comment by alicefr** [comment]
> Hey, thanks for removing all that trailing whitespace 👍
> 
> One remark inline.

Didn't actually wanted, sorry, it was by default in my editor.

File: docs/storage/disks_and_volumes.md (hunk 1 above)
Comment (alicefr) [reply docs/storage/disks_and_volumes.md]: Done, thanks!

File: docs/storage/disks_and_volumes.md [hunk 2]
```diff
@@ -2131,7 +2131,7 @@ The following strategies are supported:
   * `Replacement`: the update volumes will be replaced upon the VM restart.
   * `Migration`: the update of the volumes will trigger a storage migration of
     the old volumes to the new ones. More details about volume migration can be
-    found in the [volume migration documentation](docs/storage/volume_migration.md).
+    found in the [volume migration documentation]((/storage/volume_migration/).
```
Comment (dhiller) [inline docs/storage/disks_and_volumes.md:2134]: Apologies, I made a mistake in the suggestion. It actually requires a different style, which is then rendered out by the generator I believe.

```suggestion
    found in the [volume migration documentation](../storage/volume_migration.md).
```

Please see examples here:
https://github.com/kubevirt/user-guide/blob/dde7f54b2b4e24967f21b2f765e6e1483c60e2eb/docs/cluster_admin/confidential_computing.md?plain=1#L15-L16

File: docs/storage/disks_and_volumes.md [hunk 3]
```diff
@@ -2131,7 +2131,7 @@ The following strategies are supported:
   * `Replacement`: the update volumes will be replaced upon the VM restart.
   * `Migration`: the update of the volumes will trigger a storage migration of
     the old volumes to the new ones. More details about volume migration can be
-    found in the [volume migration documentation](docs/storage/volume_migration.md).
+    found in the [volume migration documentation]((../storage/volume_migration.md).
```
Comment (dhiller) [inline docs/storage/disks_and_volumes.md:2134]: There's one opening bracket too much here: 

[image](https://deploy-preview-821--kubevirt-user-guide.netlify.app/storage/disks_and_volumes/#update-volumes-strategy)

```suggestion
    found in the [volume migration documentation](../storage/volume_migration.md).
```

**Review by dhiller — CHANGES_REQUESTED** [review]

**Review by dhiller — APPROVED** [review]
/approve

Thank you @alicefr !

**Pull request merged**